src/autopas/tuning/tuningStrategy/fuzzyTuning/fuzzyController/LinguisticVariable.h

```diff
@@ -0,0 +1,72 @@
+/**
+ * @file LinguisticVariable.h
+ * @author Manuel Lerchner
+ * @date 18.04.24
+ */
+
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "CrispSet.h"
+#include "FuzzySet.h"
+
+namespace autopas::fuzzy_logic {
```

Comment:
Minor thing but can you rename this namespace to `FuzzyLogic` for consistency with our naming conventions, please? :D
Also, some documentation of the new namespace should go into `namespaces.h`.

Reply:
Sure. 
I followed the style used in `RuleBasedTuning` (`rule_syntax` and `autopas_generated_rule_syntax`). So we should also rename them.